src/app/cube/collection-details/collection-details.component.html

```diff
@@ -2,7 +2,7 @@
   <section class="hero">
       <div class="inner">
         <div class="logos" [ngClass]="{'long': collection?.abvName === 'ncyte' || collection?.abvName === 'cae_community' || collection?.abvName === 'nice' || collection?.abvName === 'deterlab' || collection?.abvName === '502_project' || collection?.abvName === 'ncc'}">
-          <img *ngIf="pictureLocation" src="{{ pictureLocation }}" alt="collection logo">
+          <img *ngIf="pictureLocation" src="{{ pictureLocation }}" alt="collection logo" (error)="resetPictureLocation()">
```

Comment:
```suggestion
          <img *ngIf="pictureLocation; else null" src="{{ pictureLocation }}" alt="collection logo">
```
I don't think there's a way to get this to work based on how pictureLocation is declared in the component. But in angular this is valid syntax for an ngIf